frog/imports/client/Wiki/WikiLink.js

```diff
@@ -48,7 +49,7 @@ const WikiLink = observer(
         return (
           <span
             ref={e => (this.ref = e)}
-            onClick={noFollowLinks ? null : createLinkFn}
+            onClick={noFollowLinks || preventPageCreation ? null : createLinkFn}
```

Comment:
If I understand correctly, we disable link creation entirely if page creation is disabled?

Reply:
No, only new pages cannot be clicked.